src/containers/Cheatsheet/components/Files.tsx

```diff
@@ -65,11 +65,12 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 export type FilesProps = {
   files: Array<Cheatsheet>;
-  nextPage: number | null;
-  goToNextPage: () => void;
+  hasNextPage: boolean | undefined;
+  getNextPage: () => void;
+  isLoading: boolean;
 };
 
-const Files = ({ files, nextPage, goToNextPage }: FilesProps) => {
+const Files = ({ files, hasNextPage: nextPage, getNextPage, isLoading }: FilesProps) => {
```

Comment:
Kan jo endre til hasNextPage på resten av filen også 😉 
```suggestion
const Files = ({ files, hasNextPage, getNextPage, isLoading }: FilesProps) => {
```

Reply:
haaha f2 gjorde ikke et nei lol